### ISSUE TITLE
[Quest][Zul'Drak] Troll Patrol Daily

### ISSUE THREAD

**Issue by Mistarodub** [dalaranwow/dalaran-wow#5354]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Troll Patrol daily quest in Zul'Drak sometimes doesn't award the Congratulations quest, even though I finish well ahead of required time. It just removes my "On Patrol" buff and Commander Kunz doesn't offer Congratulations quest. Happened to me twice already, so I missed out on 1000 Argent Crusade rep.

**Expected behaviour**: When you do Troll Patrol in time, Congratulations quest should always become available after you turn in Troll Patrol.

**Steps to reproduce the problem**:

1. This happens not every time, so I guess doing Troll Patrol over and over should sometimes reproduce the problem
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
https://wowgaming.altervista.org/aowow/?npc=28039
http://www.wowhead.com/spell=51573/on-patrol

Well i cannot reproduce that problem for now. Maybe someone will find the reason why the Buff "On Patrol" gets removed from time to time. You clearly need this buff in order to accept the quest "Congratulations!" (https://wowgaming.altervista.org/aowow/?quest=12604).

Just a note: If you die, the buff gets removed, as you can also see here: 
http://www.wowhead.com/spell=51573/on-patrol#comments:id=661258

> By blie (425 –  1·5)  on 2009/03/31 (Patch 3.0.9) |   |  
> The buff will disappear when you get killed. So be warned!

Setting the issue "on Hold" for now, until someone can reproduce it :)

**Comment by Mistarodub** [comment]
@Rushor Update: It seems to happen when you don't complete Alchemist's Apprentice quest on the first try and have to abandon and retake it.

**Comment by Rushor** [comment]
then the buff fades away?

**Comment by Mistarodub** [comment]
@Rushor It fades away the moment you turn in the Troll Patrol quest, making you ineligible for Congratulations reward.

**Comment by Rushor** [comment]
kay i will check
